overlord/configstate/configcore/early_test.go

```diff
@@ -38,7 +38,7 @@ func (s *earlySuite) TestEarly(c *C) {
 	patch := map[string]interface{}{
 		"experimental.parallel-instances": true,
 		"experimental.user-daemons":       true,
-		"services.ssh.disable":            true,
+		"service.ssh.disable":             true,
```

Comment:
this gets filtered so that's why we don't get an error, but it also mean that it doesn't matter too much if the left out options are real or not, we probably should fix for consistency though

Reply:
Thank you